10up/maps-block-apple
# Added support for custom marker icon.

### Description of the Change
As requested in #92, this PR adds support for the custom marker icon. Users can upload an icon image from the marker settings, Please check the following screenshot for more information. 

![Screenshot 2022-11-01 at 4 28 41 PM](https://user-images.githubusercontent.com/10613171/199219975-850fdee2-b737-47a2-a873-63abf42433b6.png)


Closes #92

### How to test the Change
1. Add Apple Maps block to the editor
2. Add a marker on the map.
3. Go to maker settings to upload icon image
4. Verify and confirm marker start showing the icon image uploaded by you (instead of default pin icon). 

### Changelog Entry
> Added - support for custom marker icon.


### Credits

Props @iamdharmesh @dinhtungdu @fabiankaegy 

### Checklist:
